### ISSUE TITLE
Custom unit does not work if not on the first value

### ISSUE THREAD

**Issue by makusnikomi** [joe223/postcss-custom-css-units#2]
# Example config

```javascript
{
    baseUnit: `--base-unit`,
    customUnit: 'rpx'
}
```

# Example input

```css
.foo {
  padding: 20rpx 40rpx 0;
}
```

# Expected output

All instances of the custom unit are transformed.

```css
.foo {
  padding: calc(var(--base-unit) * 20) calc(var(--base-unit) * 40) 0;
}
```

# Actual output

Any instances of the custom unit after the first one are left untransformed.

```css
.foo {
  padding: calc(var(--base-unit) * 20) 40rpx 0;
}
```

# Workaround

As a workaround, you can avoid this by using only the longform properties that take a single value.  For example, the following snippet is equivalent to what written above:

```css
.foo {
  padding-top: 20rpx;
  padding-right: 40rpx;
  padding-bottom: 0;
  padding-left: 40rpx;
}
```

Because each property only accepts a single value, this gets correctly transformed to the following output:

```css
.foo {
  padding-top: calc(var(--base-unit) * 20);
  padding-right: calc(var(--base-unit) * 40);
  padding-bottom: 0;
  padding-left: calc(var(--base-unit) * 40);
}
```

**Comment by makusnikomi** [comment]
Note: This also applies even if the custom unit is not used in the first value.

# Example input

```css
.foo {
  box-shadow: 0 0 20rpx #000;
}
```

# Expected output

```css
.foo {
  box-shadow: 0 0 calc(var(--base-unit) * 20) #000;
}
```

# Actual output

```css
.foo {
  /*           ⬐ missing 0             ⬐ should be 20  */
  box-shadow: 0 calc(var(--base-unit) * 0) #000;
}
```

# Workaround

To workaround this, I've used a CSS custom property, scoped to the same selector, whose value has the custom unit.  For example:

```css
.foo {
  --shadow-blur-radius: 20rpx;
  box-shadow: 0 0 var(--shadow-blur-radius) #000;
}
```

**Comment by joe223** [comment]
@makusnikomi Issue got resolved in `v0.1.0`

**Comment by makusnikomi** [comment]
Thanks @joe223 for the quick fix!
